crates/storage/store/engines/libmdbx.rs

```diff
@@ -558,7 +552,6 @@ pub fn init_db(path: Option<impl AsRef<Path>>) -> Database {
         table_info!(StateTrieNodes),
         table_info!(StorageTriesNodes),
         table_info!(CanonicalBlockHashes),
-        table_info!(Payloads),
```

Comment:
why are you removing this?

Reply:
My bad, [restored](https://github.com/lambdaclass/ethereum_rust/pull/567/commits/df1bf9b265c15d810cf608539fa6acd18bd50e37)